exchange/exchange.go

```diff
@@ -164,8 +164,8 @@ func (e *exchange) HoldAuction(ctx context.Context, r AuctionRequest, debugLog *
 
 	requestDebugInfo := getDebugInfo(r.BidRequest, requestExt)
 
-	debugInfo := requestDebugInfo && r.Account.DebugAllow
-	debugLog.Enabled = debugLog.Enabled && r.Account.DebugAllow
+	debugInfo := debugLog.DebugOverride || (requestDebugInfo && r.Account.DebugAllow)
+	debugLog.Enabled = debugLog.DebugOverride || (debugLog.Enabled && r.Account.DebugAllow)
```

Comment:
Is it necessary to factor in the override token for  `debugLog.Enabled` is there's also `DebugConsolidated`? Switching `DebugConsolidated` to a method might be a better approach here.

Reply:
Yes, modified.